roles/firmware/tasks/install.yml

```
@@ -0,0 +1,23 @@
+# check the timestamps, might want to preserve the old ones
+- name: Backup OS provided Firmware
+  copy:
+    src: "/lib/firmware/brcm/{{ item }}"
+    dest: "/lib/firmware/brcm/{{ item }}.orig"
+  with_items:
+    - brcmfmac43430-sdio.bin
+    - brcmfmac43455-sdio.bin
+
+# grab the old firmware
+- name: Retrieve older firmware
+  get_url:
+    url: "{{ item.url }}"
+    dest: "{{ item.dest }}"
+  with_items:
+    - { url: 'http://download.iiab.io/packages/brcmfmac43430-sdio.bin_2018-09-11_7.45.98.65' dest: '/lib/firmware/brcm/brcmfmac43430-sdio.bin.iiab' }
+    - { url: 'http://download.iiab.io/packages/brcmfmac43455-sdio.bin_2015-03-01_7.45.18.0_ub19.10.1' dest: '/lib/firmware/brcm/brcmfmac43455-sdio.bin.iiab' }
```

```suggestion
    - { url: 'http://download.iiab.io/packages/brcmfmac43455-sdio.bin_2015-03-01_7.45.18.0_ub19.10.1', dest: '/lib/firmware/brcm/brcmfmac43455-sdio.bin.iiab' }
```

( 3rd such line would be similar, for http://d.iiab.io/packages/brcmfmac43430-sdio.clm_blob_2018-09-11_7.45.98.65 )